src/prefect/agent/agent.py

```diff
@@ -163,7 +163,10 @@ def __init__(
 
     def _verify_token(self, token: str) -> None:
         """
-        Checks whether a token with a `RUNNER` scope was provided
+        Checks whether a token with a `RUNNER` scope was provided (DEPRECATED)
+        Visit https://docs.prefect.io/orchestration/concepts/api_keys.html
+        for information on how to use API Keys.
```

Comment:
This is a private function and is going to be confusing to a developer because this is still validating the key, right?

Reply:
Replaced this with _verify_key which does not check for RUNNER scope obviously, but may need some similar check.  Left _verify_token for backwards compatability but since it's private, I'm not sure we have to